### DISCUSSION TITLE
Set-PnPListItem when trying to update Moderation Status and other fields

### DISCUSSION THREAD

**Discussion by AtDH01** [pnp/powershell#1352] [Q&A]
Hi

This approach used to work just fine - example:

$updateListItem = @{}
$updateListItem["_ModerationStatus"] =  0 
$updateListItem["Modified"] = $ModifiedDate

$item = Set-PnPListItem -List $list -Identity 4 -Values $updateListItem -SystemUpdate

However, we now see the error: You cannot change moderation status and set other item properties at that same time.

I'm trying to update both the approval status (to approve) whilst retaining the Modified Date on the list item - this is required for a migration script.

Based on my tests, this is no longer possible as if I update just the moderation status using -UpdateType SystemUpdate, then the Modified Date is change to Now.

Is it possible to update the moderation status, without changing the modified date?

Thanks

**Comment by Dev-Mom** [comment]
Trying to update the `_ModerationStatus` property did not work for me. 
I had to use the [Set-PnPFileCheckedIn][1] cmdlet with the `-Approve` parameter. 

`Set-PnPFileCheckedIn -Url [ListItemURL] -Approve`

  [1]: https://pnp.github.io/powershell/cmdlets/Set-PnPFileCheckedIn.html
